libtrip-notification/src/test/java/com/mapbox/navigation/trip/notification/MapboxTripNotificationTest.kt

```diff
@@ -181,6 +185,8 @@ class MapboxTripNotificationTest {
         verify(exactly = 1) { bannerText.text() }
         verify(exactly = 1) { collapsedViews.setTextViewText(any(), primaryText()) }
         verify(exactly = 1) { expandedViews.setTextViewText(any(), primaryText()) }
+        verify(exactly = 1) { expandedViews.setTextViewText(any(), END_NAVIGATION) }
+        verify(exactly = 0) { expandedViews.setTextViewText(any(), STOP_SESSION) }
```

Comment:
Previously discussed

> Generally is preferable having only 1 assertion per test 👀 https://speakerdeck.com/guardiola31337/elegant-unit-testing-mobilization-2016?slide=57

Same thing applies to behavior verification tests (`verify`)